bundles/automation/org.eclipse.smarthome.automation.api/src/main/java/org/eclipse/smarthome/automation/dto/ActionDTOMapper.java

```diff
@@ -1,11 +1,10 @@
 /**
- * Copyright (c) 2016 Markus Rathgeb
+ * Copyright (c) 1997, 2016 by ProSyst Software GmbH and others.
```

Comment:
> As discussed: Our guidelines are (since the beginning) that "mvn license:format" has to be run on any PR to be accepted. That's why I say that when we have anything else in the codebase, it means that it has been merged without adhering the guidelines.

I think either this guideline is wrong, since this would mean that everyone contributing to the automation part is giving the copyright to ProSyst or which is more likely the case: The given header is simply wrong, since ProSyst simply does not have the copyright for that particular file. In this case the "and others", would be the original contributor.

An alternative to conform that guideline would be to create templates for the "mvn license:format" code generator.